modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -181,8 +336,62 @@ pattern (every view is described by several 3D-2D point correspondences).
 -   Estimate the relative position and orientation of the stereo camera "heads" and compute the
 *rectification* transformation that makes the camera optical axes parallel.
 
+<B> Homogeneous Coordinates </B><br>
```

Comment:
I added a short part on homogeneous coordinates and sprinkle in the words projective transformation, homogeneous transformation and homogeneous coordinates in the description of the camera model.